web/router.ex

```diff
@@ -18,7 +18,7 @@ defmodule TodoApi.Router do
 
     resources "/users", UserController, only: [:create]
     post "/authentication", AuthenticationController, :create
-    resources "/todos", TodoController, only: [:index]
+    resources "/todos", TodoController, except: [:new, :edit]
```

Comment:
Can we use `only` instead of `except`? Maybe it's just me, but I prefer to explicitly see what we're adding instead of what we're leaving out